pages/workspace/assets-panel-page.js

```diff
@@ -9,7 +9,7 @@ exports.AssetsPanelPage = class AssetsPanelPage extends BasePage {
     super(page);
 
     //Assets panel
-    this.assetsTab = page.locator('div[data-id="assets"]');
+    this.assetsTab = page.locator('button[data-id="assets"]');
```

Comment:
In these cases, and according to Playwright, it's preferable to [prioritize the user-facing attributes](https://playwright.dev/docs/locators) like getByRole

This other locator also makes the tests to pass and I thinks it's a bit better and less prone to future changes:
`    this.assetsTab = page.getByRole("tab", { name: "assets" });
`